OpenRA.Game/Server/Server.cs

```diff
@@ -40,8 +40,9 @@ public enum ServerState
 	public enum ServerType
 	{
 		Local = 0,
-		Multiplayer = 1,
-		Dedicated = 2
+		Skirmish = 1,
+		Multiplayer = 2,
+		Dedicated = 3
```

Comment:
then rename `Local` to `Mission` ?

Reply:
`Local` is used for more than just missions (e.g. the map editor "Play Map" option).

Reply:
perhaps then split it into Mission and Editor. We will want a save system for missions as well

Reply:
That split can be done in the PR that adds a save system for missions.